File: php-sdk/src/triplink/vcc/card/client/query/QueryCreateResultResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/7/30
 * Time: 18:01
 */
namespace triplink\vcc\card\client\query;

class QueryCreateResultResponse
{
    /**
     * query result
     * SUCCEED：成功
     * FAILURE：失败
     * EXCEPTION：异常
     */
    var $qResultStatus;

    /**
     * query result code
     */
    var $qResultCode;

    /**
     * query result desc
     */
    var $qResultMsg;

    /**
     * result of create
     * PROCESS：处理中
     * SUCCEED：成功
     * FAILURE：失败
     * EXCEPTION：异常
     */
    var $resultStatus;

    /**
     * create result code
     */
    var $resultRespCode;

    /**
     * create result desc
     */
    var $resultRespMsg;

    /**
     * channel type, TripLink distribution
     */
    var $channelType;

    /**
     * TripLink ccOpId
     */
    var $ccOpId;

    /**
     * query request number
     */
    var $requestId;

    /**
     * query request time
     */
    var $requestTime;

    /**
     * query merchant name
     */
    var $merchantName;

    /**
     * response extra info
     * json type
     * eg: {"BinFlagDcre":"12345"}
     */
    var $respExtra;

    /**
     * TripLink unique reference number
     */
    var $cardLogId;

    /**
     * card no
     * encrypt by des
     */
    var $cardNo;

    /**
     * cvv
     * encrypt by des
     */
    var $cardVerifyNo;

    /**
     * expiration date of the card
     * encrypt by des
     */
    var $cardExpiryDate;

    /**
     * @return mixed
     */
    public function getQResultStatus()
    {
        return $this->qResultStatus;
    }

    /**
     * @param mixed $qResultStatus
     */
    public function setQResultStatus($qResultStatus): void
    {
        $this->qResultStatus = $qResultStatus;
    }

    /**
     * @return mixed
     */
    public function getQResultCode()
    {
        return $this->qResultCode;
    }

    /**
     * @param mixed $qResultCode
     */
    public function setQResultCode($qResultCode): void
    {
        $this->qResultCode = $qResultCode;
    }

    /**
     * @return mixed
     */
    public function getQResultMsg()
    {
        return $this->qResultMsg;
    }

    /**
     * @param mixed $qResultMsg
     */
    public function setQResultMsg($qResultMsg): void
    {
        $this->qResultMsg = $qResultMsg;
    }

    /**
     * @return mixed
     */
    public function getResultStatus()
    {
        return $this->resultStatus;
    }

    /**
     * @param mixed $resultStatus
     */
    public function setResultStatus($resultStatus): void
    {
        $this->resultStatus = $resultStatus;
    }

    /**
     * @return mixed
     */
    public function getResultRespCode()
    {
        return $this->resultRespCode;
    }

    /**
     * @param mixed $resultRespCode
     */
    public function setResultRespCode($resultRespCode): void
    {
        $this->resultRespCode = $resultRespCode;
    }

    /**
     * @return mixed
     */
    public function getResultRespMsg()
    {
        return $this->resultRespMsg;
    }

    /**
     * @param mixed $resultRespMsg
     */
    public function setResultRespMsg($resultRespMsg): void
    {
        $this->resultRespMsg = $resultRespMsg;
    }

    /**
     * @return mixed
     */
    public function getChannelType()
    {
        return $this->channelType;
    }

    /**
     * @param mixed $channelType
     */
    public function setChannelType($channelType): void
    {
        $this->channelType = $channelType;
    }

    /**
     * @return mixed
     */
    public function getCcOpId()
    {
        return $this->ccOpId;
    }

    /**
     * @param mixed $ccOpId
     */
    public function setCcOpId($ccOpId): void
    {
        $this->ccOpId = $ccOpId;
    }

    /**
     * @return mixed
     */
    public function getRequestId()
    {
        return $this->requestId;
    }

    /**
     * @param mixed $requestId
     */
    public function setRequestId($requestId): void
    {
        $this->requestId = $requestId;
    }

    /**
     * @return mixed
     */
    public function getRequestTime()
    {
        return $this->requestTime;
    }

    /**
     * @param mixed $requestTime
     */
    public function setRequestTime($requestTime): void
    {
        $this->requestTime = $requestTime;
    }

    /**
     * @return mixed
     */
    public function getMerchantName()
    {
        return $this->merchantName;
    }

    /**
     * @param mixed $merchantName
     */
    public function setMerchantName($merchantName): void
    {
        $this->merchantName = $merchantName;
    }

    /**
     * @return mixed
     */
    public function getRespExtra()
    {
        return $this->respExtra;
    }

    /**
     * @param mixed $respExtra
     */
    public function setRespExtra($respExtra): void
    {
        $this->respExtra = $respExtra;
    }

    /**
     * @return mixed
     */
    public function getCardLogId()
    {
        return $this->cardLogId;
    }

    /**
     * @param mixed $cardLogId
     */
    public function setCardLogId($cardLogId): void
    {
        $this->cardLogId = $cardLogId;
    }

    /**
     * @return mixed
     */
    public function getCardNo()
    {
        return $this->cardNo;
    }

    /**
     * @param mixed $cardNo
     */
    public function setCardNo($cardNo): void
    {
        $this->cardNo = $cardNo;
    }

    /**
     * @return mixed
     */
    public function getCardVerifyNo()
    {
        return $this->cardVerifyNo;
    }

    /**
     * @param mixed $cardVerifyNo
     */
    public function setCardVerifyNo($cardVerifyNo): void
    {
        $this->cardVerifyNo = $cardVerifyNo;
    }

    /**
     * @return mixed
     */
    public function getCardExpiryDate()
    {
        return $this->cardExpiryDate;
    }

    /**
     * @param mixed $cardExpiryDate
     */
    public function setCardExpiryDate($cardExpiryDate): void
    {
        $this->cardExpiryDate = $cardExpiryDate;
    }



}

?>

File: vcc-sdk-java/src/main/java/com/ctrip/ccard/creditcard/vcc/bean/V1/QueryCreateResultResponse.java
package com.ctrip.ccard.creditcard.vcc.bean.V1;

/**
 * Description:
 */
public class QueryCreateResultResponse {

    /**
     * query result
     * SUCCEED：成功
     * FAILURE：失败
     * EXCEPTION：异常
     */
    private String qResultStatus;

    /**
     * query result code
     */
    private String qResultCode;

    /**
     * query result desc
     */
    private String qResultMsg;

    /**
     * result of create
     * PROCESS：处理中
     * SUCCEED：成功
     * FAILURE：失败
     * EXCEPTION：异常
     */
    private String resultStatus;

    /**
     * create result code
     */
    private String resultRespCode;

    /**
     * create result desc
     */
    private String resultRespMsg;

    /**
     * channel type, TripLink distribution
     */
    private String channelType;

    /**
     * TripLink ccOpId
     */
    private String ccOpId;

    /**
     * query request number
     */
    private String requestId;

    /**
     * query request time
     */
    private String requestTime;

    /**
     * query merchant name
     */
    private String merchantName;

    /**
     * response extra info
     * json type
     * eg: {"BinFlagDcre":"12345"}
     */
    private String respExtra;

    /**
     * TripLink unique reference number
     */
    private String cardLogId;

    /**
     * card no
     * encrypt by des
     */
    private String cardNo;

    /**
     * cvv
     * encrypt by des
     */
    private String cardVerifyNo;

    /**
     * expiration date of the card
     * encrypt by des
     */
    private String cardExpiryDate;

    public String getqResultStatus() {
        return qResultStatus;
    }

    public void setqResultStatus(String qResultStatus) {
        this.qResultStatus = qResultStatus;
    }

    public String getqResultCode() {
        return qResultCode;
    }

    public void setqResultCode(String qResultCode) {
        this.qResultCode = qResultCode;
    }

    public String getqResultMsg() {
        return qResultMsg;
    }

    public void setqResultMsg(String qResultMsg) {
        this.qResultMsg = qResultMsg;
    }

    public String getResultStatus() {
        return resultStatus;
    }

    public void setResultStatus(String resultStatus) {
        this.resultStatus = resultStatus;
    }

    public String getResultRespCode() {
        return resultRespCode;
    }

    public void setResultRespCode(String resultRespCode) {
        this.resultRespCode = resultRespCode;
    }

    public String getResultRespMsg() {
        return resultRespMsg;
    }

    public void setResultRespMsg(String resultRespMsg) {
        this.resultRespMsg = resultRespMsg;
    }

    public String getChannelType() {
        return channelType;
    }

    public void setChannelType(String channelType) {
        this.channelType = channelType;
    }

    public String getCcOpId() {
        return ccOpId;
    }

    public void setCcOpId(String ccOpId) {
        this.ccOpId = ccOpId;
    }

    public String getRequestId() {
        return requestId;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

    public String getRequestTime() {
        return requestTime;
    }

    public void setRequestTime(String requestTime) {
        this.requestTime = requestTime;
    }

    public String getMerchantName() {
        return merchantName;
    }

    public void setMerchantName(String merchantName) {
        this.merchantName = merchantName;
    }

    public String getRespExtra() {
        return respExtra;
    }

    public void setRespExtra(String respExtra) {
        this.respExtra = respExtra;
    }

    public String getCardLogId() {
        return cardLogId;
    }

    public void setCardLogId(String cardLogId) {
        this.cardLogId = cardLogId;
    }

    public String getCardNo() {
        return cardNo;
    }

    public void setCardNo(String cardNo) {
        this.cardNo = cardNo;
    }

    public String getCardVerifyNo() {
        return cardVerifyNo;
    }

    public void setCardVerifyNo(String cardVerifyNo) {
        this.cardVerifyNo = cardVerifyNo;
    }

    public String getCardExpiryDate() {
        return cardExpiryDate;
    }

    public void setCardExpiryDate(String cardExpiryDate) {
        this.cardExpiryDate = cardExpiryDate;
    }
}
